Makefile

```diff
@@ -472,12 +463,34 @@ endif
 	@ecs-cli compose -f docker-compose.prod.yml up
 
 
-
-# Deprecated targets
 deploy:
-	@echo $@ ERROR: see deployment steps for accelerate.
-	@echo see: https://github.com/masschallenge/standards/blob/376d290b41a202acc5c2263d7275ba4a57330ad7/create_new_release.md#deploy-to-staging
+	@if [ "$$IMAGE_TAG" != "development" ] && [ "$$IMAGE_TAG" != "AC-6678" ]; then exit 1; fi;
+	@pip install --upgrade certifi pyopenssl requests[security] ndg-httpsclient pyasn1 pip botocore
+	@curl -s https://raw.githubusercontent.com/silinternational/ecs-deploy/master/ecs-deploy | sudo tee /usr/bin/ecs-deploy
+	@sudo chmod +x /usr/bin/ecs-deploy
+	@git clone https://$$GITHUB_ACCESS_TOKEN@github.com/masschallenge/infrastructure -b pre-staging
+	@wget https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_linux_amd64.zip -q
+	@sudo unzip terraform_0.11.11_linux_amd64.zip -d /usr/bin/
+	@sudo echo "deb http://us.archive.ubuntu.com/ubuntu vivid main universe" >> /etc/apt/sources.list
+	@cd infrastructure/environment/test/static-secrets; \
+	terraform init; \
```

Comment:
@VincentHokie why do we need terraform? My understanding was that Terraform is used for infrastructure automation and with that in mind, this doesn't seem to be the right application.
Can't we trigger some sort of refresh with just the `aws cli`?

Reply:
there's a cli trigger that is done [here](https://github.com/masschallenge/impact-api/pull/294/files#diff-b67911656ef5d18c4ae36cb6741b7965R487) incase the infrastructure changes themselves don't lead to a deploy.

if e.g. we edit our [task definition](https://github.com/masschallenge/infrastructure/blob/development/super_stack/ecs-cluster/impact/task-def.json) (with say a new secret), terraform would do 2 things.
- update the infrastructure &
- perform a deploy on ECS

Adding terraform here has the benefit of us being able to test continuous changes to our infrastructure as well.

Reply:
In the spirit of separation of concerns. Would it not make more sense to have the terraform on the infrastructure repo and keeping this to the simple task of triggering a deploy? 
The simpler, the easier it will be to maintain by people without terraform experience.

Reply:
sure, makes sense. done.

Reply:
note though, that were going to have several stacks up built off e.g master. So a single travis build on the infrastructure repo updating > 1 AWS Stack won't be an easy thing to accomplish.

Reply:
Do we really need to have continuous deployment on infrastructure? A bug in Accelerate code might not be as disruptive as one in the Infrastructure code. I wonder what the best practices are in the infrastructure as code community.

Reply:
I think we can hold off on CD for infra for now.